src/wrapper.js

```diff
@@ -34,8 +34,6 @@ class Wrapper {
         const { region, table, kmsKey } = this.options;
         if(region && table)
             this.store = new SecretStore({ table, awsOpts: { region }, kmsKey });
-        else
```

Comment:
When would there be a situation in which the region and table aren't defined, but in which we can still read a secret?

Reply:
Yeah, in local development.

Reply:
Which in that case you override the yadda environment in your docker-compose. This is an intentionally thrown error for when you're in prod with a bad configuration. Effectively all errors return in null, which in a config _may_ be a correct value but now have no idea if it's a default value provided by this or from the secret stash.

Reply:
This is more for the ones that are specified with KMSVAR(...) in the docker.js config. Since there's no process.env variable, you can't override it.

Reply:
I thought KMSVAR automatically pulls from the environment if it can't find a matching secret.  I agree with Chris, simply swallowing errors in all environments doesn't seem like what we actually want.  There should definitely be a way to use this is in dev, though.

Reply:
Yeah, it pulls from the env first, then any override you have set, and then dynamodb. However, this error is when you don't have a region and table defined, which is the case when we are doing local dev.